apps/src/aiComponentLibrary/userMessageEditor/UserMessageEditor.tsx

```diff
@@ -56,7 +57,18 @@ const UserMessageEditor = React.forwardRef<
       [onSubmit]
     );
 
+    useEffect(() => {
+      if (!internalInputRef.current) {
+        return;
+      }
+
+      internalInputRef.current.style.height = 'auto'; // Need to reset height before update.
+      internalInputRef.current.style.height =
+        internalInputRef.current.scrollHeight + 2 + 'px';
```

Comment:
nit: quick comment describing why this is needed?

Reply:
Added -- without it, there end up being scrollbars on the textarea. I'm not 100% sure if it's a "pixel rounding" problem or if there's a less janky way I could solve this, but seems to do the trick.